### ISSUE TITLE
Connection broken or TokenExpiredError for certain cube requests 

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube-sh#49]
When requesting large cubes via the sh store, it happens, that the connection breaks or the token expires. Therefore the generated cube is not complete and useless. 

To reproduce see issue https://github.com/bcdev/service-eurodatacube/issues/132

**Comment by pont-us** [comment]
The "Connection broken" error

```
requests.exceptions.ChunkedEncodingError: ("Connection broken: InvalidChunkLength(got length b'', 0 bytes read)", InvalidChunkLength(got length b'', 0 bytes read))
```

has now been fixed by PR #79 and successfully tested with a few hundred GB of AVL data. The `TokenExpiredError` persists. The suggested workaround is to split the required data into shorter time-spans, generate the corresponding sub-datasets with separate invocations of xcube_sh, and concatenate them with nc2zarr .

**Comment by forman** [comment]
Seemed solved after #82, but now we get 401 from Sentinel Hub - weird,

**Comment by forman** [comment]
Closed by #83
